nni/tools/nnictl/command_utils.py

```diff
@@ -31,14 +32,25 @@ def check_output_command(file_path, head=None, tail=None):
         exit(1)
 
 
-def kill_command(pid):
-    """kill command"""
+def kill_command(pid, timeout=60):
+    """Kill the process of pid (with a terminate signal).
+    Waiting up to 60 seconds until the process is killed.
+    """
+    # TODO: The input argument should better be Popen rather than pid.
     if sys.platform == 'win32':
-        process = psutil.Process(pid=pid)
-        process.send_signal(signal.CTRL_BREAK_EVENT)
+        try:
+            process = psutil.Process(pid=pid)
+            process.send_signal(signal.CTRL_BREAK_EVENT)
```

Comment:
I don't know why not using `process.terminate()` or `process.kill()` here.
But well, not your fault.

Reply:
I don't know either why this is pid here instead of process.

I noticed the usage of psutil is limited to Windows. It reminds that someone once told me that psutil is something binary, and we plan to drop it in future.